client/state/sites/plans/selectors/get-plan-discounted-raw-price.js

```diff
@@ -19,10 +19,6 @@ export function getPlanDiscountedRawPrice(
 ) {
 	const plan = getSitePlan( state, siteId, productSlug );
 
-	if ( ( plan?.introductoryOfferRawPrice ?? -1 ) > 0 ) {
```

Comment:
We are going to rely on `getIntroOfferPrice`, and here we are going to stop returning the introductory offer price since we need the "normal" price to fall back on if the user is ineligible for the intro offer.